.github/workflows/check-catalyst.yaml

```diff
@@ -526,6 +526,11 @@ jobs:
     - name: Checkout Catalyst repo
       uses: actions/checkout@v4
 
+    - name: Install device dependencies (OpenQasm device)
```

Comment:
Just out of curiosity, what was the reason behind this relocation?

Reply:
The OpenQASM device explicitly depends on a released PL version. If it appears as the last step in the CI tree, it overrides the pinned PennyLane version with a released version. This will ensure the version in the dep file is the last one installed.